lib/foreman_deployments/engine.rb

```diff
@@ -4,10 +4,7 @@
 
 module ForemanDeployments
   class Engine < ::Rails::Engine
-    config.autoload_paths += Dir["#{config.root}/app/controllers/foreman_deployments/concerns"]
-    config.autoload_paths += Dir["#{config.root}/app/helpers/foreman_deployments/concerns"]
-    config.autoload_paths += Dir["#{config.root}/app/models/foreman_deployments/concerns"]
-    config.autoload_paths += Dir["#{config.root}/app/overrides"]
+    config.autoload_paths += Dir["#{config.root}/app/**/"]
```

Comment:
according to here: http://stackoverflow.com/a/10167672
`autoload_paths` does not support glob patterns, although didn't try it myself yet.